tutorial/src/main/kotlin/com/kaspersky/kaspresso/tutorial/flaky/FlakyActivity.kt

```diff
@@ -0,0 +1,53 @@
+package com.kaspersky.kaspresso.tutorial.flaky
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import com.kaspersky.kaspresso.tutorial.databinding.FlakyActivityBinding
+import java.util.concurrent.TimeUnit
+
+class FlakyActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val binding = FlakyActivityBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        binding.sixthElementBtn.setOnClickListener {
+            startActivity(Intent(this, ResultActivity::class.java))
+        }
+
+        Handler(mainLooper)
```

Comment:
Coroutines maybe?

Reply:
I removed this FlakyActivity because it's unused for now. Later we will create a lesson for it and will add new FlakyActivity with coroutines